frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -9,56 +9,48 @@ import { WINTOOLS_CONTAINER_NAMES } from '../modals/cdrom-vm-modal/constants';
 import { VMKind } from '../../types';
 import { V1Disk } from '../../types/vm/disk/V1Disk';
 
-const DiskSummaryRow: React.FC<DiskSummaryRowProps> = ({ title, value }) => (
-  <div className="kubevirt-disk-summary__disk">
-    <div id={`kubevirt-disk-summary-disk-title-${title}`}>{title}</div>
-    <div id={`kubevirt-disk-summary-disk-value-${value}`}>{value}</div>
-  </div>
-);
+import './disk-summary.scss';
 
 export const DiskSummary: React.FC<DiskSummaryProps> = ({ disks, vm }) => (
-  <div className="kubevirt-disk-summary">
-    {disks.map(({ name }, i) => {
+  <dl className="oc-vm-details__datalist kubevirt-disk-summary">
+    {disks.map(({ name }) => {
       const container = getContainerImageByDisk(vm, name);
       const pvc = getPVCSourceByDisk(vm, name);
       const url = getURLSourceByDisk(vm, name);
+      let value = '';
 
       if (_.includes(WINTOOLS_CONTAINER_NAMES, container)) {
-        return (
-          <DiskSummaryRow
-            key={`disk-summary-${name}`}
-            title={`Drive ${i + 1}: Windows Tools`}
-            value={container}
-          />
-        );
-      }
-      if (container) {
-        return (
-          <DiskSummaryRow
-            key={`disk-summary-${name}`}
-            title={`Drive ${i + 1}: Container`}
-            value={container}
-          />
-        );
-      }
-      if (url) {
-        return (
-          <DiskSummaryRow key={`disk-summary-${name}`} title={`Drive ${i + 1}: URL`} value={url} />
-        );
+        value = `Windows Tools: ${container}`;
+      } else if (container) {
+        value = `Container: ${container}`;
+      } else if (url) {
+        value = `URL: ${url}`;
+      } else if (pvc) {
+        value = `PVC: ${pvc}`;
       }
+
+      const nameKey = `kubevirt-disk-summary-disk-title-${name}`;
+      const valueKey = `kubevirt-disk-summary-disk-title-${value}`;
```

Comment:
nit: considering the `value` contains space, wouldn't it make more sense to recycle the `name` even here?